UTIL/wrfcmaq_twoway_coupler/README.md

```diff
@@ -1,10 +1,10 @@
 # WRF-CMAQ Coupler: 
 
-The new WRF-CMAQ model is based on WRF 4.3 and CMAQ 5.3.3 and supports the RRTMG radiation scheme for short wave aerosol direct effect. The new version uses a core-shell representation to perform aerosol optics calculations rather than the volume mixing technique applied in the previous versions of WRF-CMAQ. )
+The new WRF-CMAQ model is based on WRF 4.4 and CMAQ 5.3.3 (and later) and supports the RRTMG radiation scheme for short wave aerosol direct effect. The new version uses a core-shell representation to perform aerosol optics calculations rather than the volume mixing technique applied in the previous versions of WRF-CMAQ. )
```

Comment:
Again "v5.3.3+" and later...

Reply:
I deleted the wrfcmaq_twoway_coupler folder.